src/nvim/menu.c

```diff
@@ -1156,14 +1154,18 @@ static char_u *popup_mode_name(char_u *name, int idx)
 }
 
 
-/*
- * Duplicate the menu item text and then process to see if a mnemonic key
- * and/or accelerator text has been identified.
- * Returns a pointer to allocated memory, or NULL for failure.
- * If mnemonic != NULL, *mnemonic is set to the character after the first '&'.
- * If actext != NULL, *actext is set to the text after the first TAB.
- */
+/// Duplicate the menu item text and then process to see if a mnemonic key
+/// and/or accelerator text has been identified.
+///
+/// @param str
```

Comment:
Honest question, what's the point of this being here if it doesn't have a description? I understand that the purpose is mostly obvious, but it would be more helpful if there was just a short snippet describing it, like for the other parameters. For example, [this Doxygen page] provides descriptions for some parameters, but many are left out for no obvious reason.